src/Plugin/KeyInput/ApigeeAuthKeyInput.php

```diff
@@ -134,14 +134,24 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
         '#type' => 'checkbox',
         '#title' => $this->t('Use the default service account if this portal is hosted on GCP'),
         '#description' => $this->t("Please ensure you have added 'Apigee Developer Administrator' role to the default compute engine service account hosting this portal."),
-        '#default_value' => $values['gcp_hosted'] ?? TRUE,
+        '#default_value' => $values['gcp_hosted'] ?? FALSE,
         '#states' => [
           'visible' => $state_for_hybrid,
         ],
       ];
       $state_for_not_gcp_hosted = [
         ':input[name="key_input_settings[gcp_hosted]"]' => ['checked' => FALSE],
       ];
+      if (empty($form['organization']['#default_value'])) {
+        $organizations = $gceServiceAccountAuth->getAuthorizedOrganizations();
+        $organization = array_pop($organizations);
```

Comment:
What about the other orgs? What about the other available? (The method name indicates to me that there could be more). Why not just `$organization = reset($organizations);`?